subprojects/docs/src/snippets/testKit/gradleVersion/groovy/build.gradle

```diff
@@ -11,6 +11,8 @@ dependencies {
     testImplementation('org.spockframework:spock-core:2.2-groovy-3.0') {
         exclude group: 'org.codehaus.groovy'
     }
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.10.0'
+    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
```

Comment:
❔ Why was this change necessary?

Reply:
because of this: https://docs.gradle.org/current/userguide/upgrading_version_8.html#manually_declaring_dependencies

Why I'm the one noticing that is a different interesting question tough.

Reply:
Was the snippet test failing without your changes?

Reply:
The test is also failing on master locally and is not executed in CI because of https://github.com/gradle/gradle/blob/a773afd64882f5b28ae24f3df3eeb4fa9765b2e3/subprojects/docs/build.gradle#L642C16-L642C16 and that `:docs:docsTest` in CI is only running on Java 20?

@eskatos Do you think I should back this change out into a separate PR?